### ISSUE TITLE
WebServices: GlobalWebServicesHandler SPI runtime not available for Jakarta EE9

### ISSUE THREAD

**Issue by jagraj** [OpenLiberty/open-liberty#18187]
**Describe the bug**
Our application has Liberty user feature which is consuming GlobalWebServicesHandler SPI feature and our application is transformed to Jakarta EE9 runtime and we can not build our liberty user feature as we do not have SPI runtime not available in beta drivers.

![image](https://user-images.githubusercontent.com/713849/129390328-1c209c6e-1db5-4114-9ee9-0379d5a2dbc8.png)

**Steps to Reproduce**
1. Install Liberty beta runtime
2. GlobalWebServicesHandler liberty user feature fails to build after jakarta transformation

**Expected behavior**
We should be able to build and run with the latest beta runtime once we have SPI jar available.

**Diagnostic information:**
 - OpenLiberty Version: Open Liberty 21.0.0.9-beta/wlp-1.0.55.cl210820210727-1323
 - Java Version: OpenJDK 64-Bit Server VM, version 1.8.0_292-b10
 - server.xml configuration (WITHOUT sensitive information like passwords)
 - If it would be useful, upload the messages.log file found in `$WLP_OUTPUT_DIR/messages.log`

**Additional context**

**Comment by jhanders34** [comment]
Resolved with the changes for #11497

**Comment by jagraj** [comment]
I got latest daily build and switched my liberty user feature import for Global WebServices handler to Jakarta version 10.0.0 and my application works fine now. Thanks for the fix. 

Here is my MANIFEST.MF file...

```
Manifest-Version: 1.0
Bundle-SymbolicName: WASPersonaWebServicesHandler
Export-Package: com.ibm.wsspi.webservices.persona.api,com.ibm.wsspi.we
 bservices.persona.handler;uses:="com.ibm.wsspi.webservices.handler,or
 g.osgi.framework,com.ibm.wsspi.webservices.persona.api"
Bundle-Name: WASPersonaWebServicesHandler
Bundle-Version: 1.0.0.202110181027
Bundle-ManifestVersion: 2
Bundle-Activator: com.ibm.wsspi.webservices.persona.handler.WASPersona
 WebServicesHandlerActivator
Bundle-RequiredExecutionEnvironment: JavaSE-1.7,
 JavaSE-1.8
Import-Package: com.ibm.json.java;version="1.0.0",
 com.ibm.json.xml;version="1.0.0",
 com.ibm.wsspi.webservices.handler;version="10.0.0",
 jakarta.servlet;version="5.0.0",
 jakarta.servlet.http;version="5.0.0",
 jakarta.xml.ws;version="3.0.0",
 jakarta.xml.ws.handler;version="3.0.0",
 org.osgi.framework

```
